packages/upload-client/src/types.ts

```diff
@@ -45,6 +45,16 @@ import {
   UsageReportFailure,
 } from '@web3-storage/capabilities/types'
 
+type Override<T, R> = Omit<T, keyof R> & R
+
+type FetchOptions = Override<
+  IpfsUtilsFetchOptions,
+  {
+    // `fetch` is a browser API and browsers don't have `Readable`
+    body: Exclude<IpfsUtilsFetchOptions['body'], import('node:stream').Readable>
```

Comment:
What does this do?

Reply:
it excludes `Readable` from the allowed body types that come from `ipfs-utils` because that's not what other `fetches` will expect